config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql

```diff
@@ -0,0 +1,84 @@
+PREFIX meb:     <http://rdf.myexperiment.org/ontologies/base/>
+PREFIX adms:    <http://www.w3.org/ns/adms#>
+PREFIX prov:    <http://www.w3.org/ns/prov#>
+PREFIX ext:     <http://mu.semte.ch/vocabularies/ext/>
+PREFIX eli:     <http://data.europa.eu/eli/ontology#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX pav:     <http://purl.org/pav/>
+prefix mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX schema:  <http://schema.org/>
+PREFIX dct:     <http://purl.org/dc/terms/>
+
+INSERT {
+  GRAPH ?h {
+    ?sub schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?form schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?decisionType schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?submissionDocument schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?submissionDocumentStatus schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?authenticityType schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?taxType schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?chartOfAccount schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?file schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?submissionFile schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?submissionDocumentFile schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?formDataFile schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+  }
+  GRAPH ?j {
+    ?concept schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+  }
+}
+WHERE {
+  GRAPH ?g {
+    ?sub a meb:Submission ;
+        adms:status <http://lblod.data.gift/concepts/9bd8d86d-bb10-4456-a84e-91e9507c374c> ; # SENT
+        prov:generated ?form ;
+        pav:createdBy ?bestuurseenheid ;
+        dct:subject ?submissionDocument .
+
+
+    ?form <http://lblod.data.gift/vocabularies/besluit/authenticityType> |
```

Comment:
here you could make a separate list of types that need to be exported. That's a one time check you can do, not to check for every separate sbumission. (the walking of this tree is super expensive)